README.md

```diff
@@ -2,18 +2,16 @@
 
 ![Wizards Logo](wizards-logo.jpg)
 
-*****
 
-*Copyright (C) 2017 Improbable Worlds Limited. All rights reserved.*
 
-- [Guide](https://docs.improbable.io/reference/13.0/shared/get-started/tour)
-- *GitHub repository*: https://github.com/spatialos/Wizards/
+- [Guide](https://docs.improbable.io/reference/13.0/shared/get-started/tour) (Website docs)
+- *GitHub repository*: github.com/spatialos/Wizards
```

Comment:
This may need to be using `[]` to be turned into a link, similar to https://github.com/spatialos/UnityStarterProject/pull/9/files#diff-04c6e90faac2675aa89e2176d2eec7d8R4